### PR TITLE
ML2-187 Starting point for promo data in checkout

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

File: src/components/Checkout/LoanPrice.vue [hunk 1]
```diff
@@ -156,6 +156,7 @@ export default {
 								numeral(updatedPrice).value()
 							);
 							this.$emit('refreshtotals', this.changeType === 'remove' ? 'removeLoan' : '');
+							this.$emit('updating-totals', false);
```
Comment (mcstover) [inline src/components/Checkout/LoanPrice.vue:159]: Ensures the competed state is emitted.

File: src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue [hunk 2]
```diff
@@ -350,12 +351,25 @@ const basketItemsQuery = gql`query basketItemsQuery(
...
+				kivaCardTotal
+				kivaCreditAvailableTotal
+				kivaCreditAppliedTotal
+				kivaCreditRemaining
+				kivaCreditToReapply
 				loanReservationTotal
+				redemptionCodeAppliedTotal
+				redemptionCodeAvailableTotal
+				universalCodeAppliedTotal
+				universalCodeAvailableTotal
```
Comment (mcstover) [inline src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue:372]: I'll be doing cleanup passes later to move this entire inline query out into it's own file and use the shopTotals fragment.

File: src/components/Checkout/OrderTotals.vue [hunk 3]
```diff
@@ -1,6 +1,10 @@
 <template>
-	<div class="order-totals row">
-		<div v-if="showKivaCredit" class="kiva-credit columns small-12">
+	<div class="order-totals">
+		<div class="order-total" v-if="showPromoCreditTotal">
+			<strong>Order Total: <span class="total-value">{{ itemTotal }}</span></strong>
+		</div>
```
Comment (mcstover) [inline src/components/Checkout/OrderTotals.vue:5]: This a is a new block we could potentially show for standard checkout, however it's Promo only for now.

File: src/components/Checkout/OrderTotals.vue [hunk 4]
```diff
@@ -22,9 +26,81 @@
...
+				{{ promoFundDisplayDescription }}
+			</kv-tooltip>
+		</div>
+
+		<div class="order-total">
+			<strong>
+				<template v-if="!showPromoCreditTotal">Total: </template>
+				<template v-else>Total Due: </template>
+				<span class="total-value">{{ creditAmountNeeded }}</span>
+			</strong>
```
Comment (mcstover) [inline src/components/Checkout/OrderTotals.vue:71]: This is our original and permanent Total line item with an adjust value variable name and copy clarifications in the promo campaign view.

File: src/components/Checkout/OrderTotals.vue (hunk 3 above)
Comment (emuvente) [reply src/components/Checkout/OrderTotals.vue:5]: 👏

File: src/components/Checkout/OrderTotals.vue (hunk 4 above)
Comment (emuvente) [reply src/components/Checkout/OrderTotals.vue:71]: 👏 
I feel that we've needed something like this (in combination with that Order Total line) in regular checkout for quite some time now, so I'm really glad to see it making an appearance here!

**Review by emuvente — APPROVED** [review]
👍

**Comment by BoulderBrains** [comment]
👀 👍 
This is awesome, getting closer and closer to leaving the old checkout page/flow behind!

**Pull request merged**